packages/search/test/users/users.test.ts

```diff
@@ -0,0 +1,397 @@
+/* Copyright (c) 2021 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { UserSession } from "@esri/arcgis-rest-auth";
+import {
+  UserService,
+  ISearchUsersFilter,
+  UserSortableField,
+  SortDirection,
+  GraphQLError
+} from "../../src";
+import {
+  createSessionMutation,
+  userSelfQuery,
+  userSearchQuery
+} from "../../src/users/queries";
+import { GraphQLClient } from "graphql-request";
+
+describe("user service", () => {
+  const portalUrl = "http://arcgis.com/sharing/rest";
+
+  it("static constructor", () => {
+    const session: UserSession = new UserSession({
+      token: "token"
+    });
+    const service = UserService.create(
+      portalUrl,
+      "https://foo.com/graphql",
+      session
+    );
+    expect(service).toBeTruthy();
```

Comment:
Not sure how else to test this. We can't access the created GraphQLClient object to check that the token was properly set or anything because it is private.